### PR TITLE
add jump to latest messages button

### PR DESCRIPTION
**Pull request by khubo**
jumpTo

### CODE REVIEW

**Comment by khubo** [comment]
TODO:
  1.) fix lint in style.scss

File: app/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import thunk from 'redux-thunk'
 
 const store = createStore(
   reducer,
-  compose(applyMiddleware(thunk, logger))
+  compose(applyMiddleware(thunk))
```
Comment (khubo) [inline app/index.js]: restore this.

**Comment by khubo** [comment]
@nikolaiwarner i am confused about the scss linting.. :neutral_face: . suggestions?

**Comment by nikolaiwarner** [comment]
@khubo what if we dont stress about the scss lint for now and we swiftly move to styled-components?

**Comment by khubo** [comment]
@nikolaiwarner that will be awesome! let's do it.

**Comment by nikolaiwarner** [comment]
@khubo i started adding in styled-components over here: https://github.com/cabal-club/cabal-ui/tree/master/react

i like it a lot. i've converted over the Message component and Avatar component to use it. Check it out and let me know what you think. If you're interested, maybe we can split up the work somehow.

As for this branch, I've been having trouble with timely syncing of messages that its been hard to test getting new messages. :| I'll keep trying though!

**Comment by khubo** [comment]
@nikolaiwarner I havent added a tag for new messages in the PR. Its only jump to latest.

Checked that repo for styled-components. It's awesome that there is storyboard supported added. I do have some time now, and will slowly contribute to it. Btw, do you think a monorepo setup would be good here?

**Comment by nikolaiwarner** [comment]
hi @khubo! I made a few updates to this, I hope you dont mind! I think its ready to merge in. What do you think?

**Comment by khubo** [comment]
@nikolaiwarner it looks much better now!! plis merge.

**Pull request merged**